src/config/ormconfig.ts

```diff
@@ -6,10 +6,10 @@ const isProduction =
 export const databaseConfig: DataSourceOptions = {
   type: 'postgres',
   host: process.env.DB_HOST || 'localhost',
-  port: parseInt(process.env.DB_PORT, 10) || 5432,
+  port: parseInt(process.env.DB_PORT, 10) || 5435,
```

Comment:
This will be reverted. 
Waiting on @MHA10 's response on #2 

I wasn't able to read these variables from .env.